requirements-dev.txt

```diff
@@ -25,9 +25,6 @@ pillow
 pre-commit
 pycodestyle
 pytest
-pytest-cov
-pytest-flake8
-pytest-xdist
```

Comment:
The only advantage for keeping this is for local use but I'd still remove it from our CIs.